daisy_workflows/export_metadata/export-metadata.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+
+"""Export the image metadata on a GCE VM.
+
+Parameters retrieved from instance metadata:
+
+google_cloud_repo: The repo to use to build image. Must be one of
+  ['stable' (default), 'unstable', 'staging'].
+image_tar_location: The Cloud Storage destination for the resultant image.
+metadata_dest: The Cloud Storage destination for the resultant image metadata.
+image_family: The family that image belongs.
+distribution: Use image distribution. Must be one of [enterprise_linux,
+  debian, centos].
+uefi: boolean Whether using UEFI to boot OS.
+"""
+
+import datetime
+import json
+import logging
+import re
+import subprocess
+import sys
+import tempfile
+import urllib.error
+import urllib.request
+
+from google.cloud import exceptions
+from google.cloud import storage
+
+METADATA_ENDPOINT = 'http://metadata.google.internal/computeMetadata/v1' \
+                    '/instance/attributes/?recursive=true '
+
+
+def GetMetadataAttribute():
```

Comment:
can we use the shared library in this repo, which i think you've copied these functions from?

Reply:
Sure. I am going to use the function in common.py